lib/yoti/http/profile_request.rb

```diff
@@ -15,6 +15,9 @@ def receipt
 
     def request
       yoti_request = Yoti::Request.new
+      yoti_request.add_header('X-Yoti-Auth-Key', Yoti::SSL.auth_key_from_pem)
+      yoti_request.add_header('Content-Type', 'application/json')
```

Comment:
We don't need the `Content-Type` for this request. Generally you only need the content-type for PUT or POST requests (when there's a payload), and this is the case for the profile request too.

Reply:
What about the Accept?

Reply:
It's not necessary for this I believe, so we can omit.